### ISSUE TITLE
Merging `tasks.*.json` to `tasks.json` only merges `tasks.local.json`

### ISSUE THREAD

**Issue by agreedSkiing** [swellaby/vscode-workspace-config-plus#111]
As the title of this issue for some reason the merging of tasks only merges the local.json file and not both.

#### Plugin
```text
Name: Workspace Config+
Id: Swellaby.workspace-config-plus
Description: VS Code extension for improved workspace configuration
Version: 0.2.4
Publisher: Swellaby
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=swellaby.workspace-config-plus
```
#### VSCode
```text
Version: 1.75.1 (user setup)
Commit: 441438abd1ac652551dbe4d408dfcec8a499b8bf
Date: 2023-02-08T21:32:34.589Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No
```

#### tasks.shared.json
```json
{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "label": "server-sonar",
            "type": "shell",
            "command": "${workspaceFolder}/deployment/podman/run_sonarqube.sh",
            "problemMatcher": []
        },
        {
            "label": "scan-sonar",
            "type": "shell",
            "command": "${workspaceFolder}/deployment/podman/run_scan_sonar.sh",
            "problemMatcher": []
        },
        {
            "label": "test-pipeline",
            "type": "shell",
            "command": "${workspaceFolder}/deployment/podman/run_all_test_pipeline.sh",
            "problemMatcher": []
        },
        {
            "label": "test-all",
            "type": "shell",
            "command": "${workspaceFolder}/deployment/podman/run_all_test.sh",
            "problemMatcher": []
        },
        {
            "label": "test-maven",
            "type": "shell",
            "command": "mvn clean test",
            "problemMatcher": []
        },
        {
            "label": "lint-java",
            "type": "shell",
            "command": "npx prettier --check \"src/**/*.java\"",
            "problemMatcher": []
        }
    ]
}
```

#### tasks.local.json
```json
{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "label": "merge-develop",
            "type": "shell",
            "command": "git merge origin/develop",
            "problemMatcher": []
        },
        {
            "label": "lint-all",
            "type": "shell",
            "command": "npx prettier --check .",
            "problemMatcher": []
        }
    ]
}
```
### tasks.json
```
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "merge-develop",
      "type": "shell",
      "command": "git merge origin/develop",
      "problemMatcher": []
    },
    {
      "label": "lint-all",
      "type": "shell",
      "command": "npx prettier --check .",
      "problemMatcher": []
    }
  ]
}
```

### Tests
I update `tasks.local.json` with in the array of tasks
```json
    {
      "label": "pull",
      "type": "shell",
      "command": "git pull",
      "problemMatcher": []
    }
```
which produceses on save `[2023-03-01 09:10:40] [INFO] Updating config in /home/eriase/repos/quarkus-maven-project/.vscode/tasks.json`.

After this I remove the added line from `tasks.local.json` the same log is produced and add it to update the tasks.shared.json and no log is produced at all ☹

If I add an incomplete json object like below the it failes to parse the `tasks.shared.json` file as excepted.
```json
       {
            "label":
        }
```
```log
[2023-03-01 09:18:54] [ERROR] Failed to parse contents of: /home/eriase/repos/quarkus-maven-project/.vscode/tasks.shared.json
[2023-03-01 09:18:54] [DEBUG] Error: Failed to parse contents of: /home/eriase/repos/quarkus-maven-project/.vscode/tasks.shared.json
```

**Comment by witcher112** [comment]
Resolved by #107 , we're waiting for @calebcartwright to update the extension on marketplace

**Comment by calebcartwright** [comment]
deep merge avail in v0.2.5
